### ISSUE TITLE
Manage internationalization

### ISSUE THREAD

**Issue by Djamel-Ali** [ardovic/Open-Source-Android-Weather-App#52]
### Description

Each contributor can translate the GUI into his/her mother tongue (or any other language of his/her choice, but "that he/she masters" ).

### Steps to reproduce
The strings.xml file is made for example to facilitate the internationalization task, so most (maybe all) of the translation (i.e. I18N) will be done in this file.

### Expected behavior
After choosing a language in the application settings, the entire GUI will be loaded in this language.

### Actual behavior
The GUI is currently only in English.

### Who will work on this issue ?
If there's no one to translate it into French or Kabyle (Tamazight), I could do that.
Each contributor who would like to work on this issue, just create one issue per language and/or culture.
You can mention this issue in your Commit Message (if all of you agree), by adding #52 in the form written by @ardovic (owner) in the readme file of this project.

**Comment by ardovic** [comment]
Good idea, localization would be beneficial indeed.

**Comment by SumaraNihal** [comment]
@ardovic  Is this issue still open to contribute?

**Comment by ardovic** [comment]
@SumaraNihal Yes, there are many languages missing at the moment, feel free to open PR

**Comment by pradeep14012004** [comment]
Hi @ardovic, I would like to add localization support for French.
I will submit a PR with translated strings.xml.
